code/game/objects/items/holotool.dm

```diff
@@ -0,0 +1,83 @@
+
+
+/obj/item/holotool
+	name = "experimental holotool"
+	desc = "A highly experimental holographic tool projector."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "holotool"
+	slot_flags = SLOT_BELT
+	usesound = 'sound/items/pshoom.ogg'
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	actions_types = list(/datum/action/item_action/change_tool, /datum/action/item_action/change_ht_color)
+
+	var/on = FALSE
+	var/obj/item/current_tool = 0
+	var/current_color = "#48D1CC" //mediumturquoise
+
+/obj/item/holotool/proc/AddTool(typee, namee)
+	var/obj/item/WR = new typee(src)
+	WR.forceMove(src)
+	WR.name = namee
+	WR.usesound = usesound //use the same sound as we do
+	WR.toolspeed = 0.5
```

Comment:
Would change this to 1.25 to alleviate powercreep concerns, its a huge space saving compared to a normal set of tools.